pixi.toml

```diff
@@ -0,0 +1,95 @@
+[project]
+authors = [
+  "Albert Steppi <albert.steppi@gmail.com>",
+  "Irwin Zaid <irwin.zaid@gmail.com>",
+]
+channels = ["https://prefix.dev/conda-forge"]
+description = "Special function implementations."
+name = "xsf"
+platforms = ["win-64", "linux-64", "osx-64", "osx-arm64"]
+
+## Build
+
+[feature.build.dependencies]
+cmake = ">=3.30.5,<4"
+cxx-compiler = ">=1.8.0,<2"
+make = ">=4.4.1,<5"
+
+[feature.build.tasks.configure]
+cmd = [
+  "cmake",
+  # The source is in the root directory
+  "-S .",
+  # We want to build in the build directory
+  "-B build",
+]
+cwd = "."
+
+[feature.build.tasks.build-only]
+# Just build without configure
+cmd = ["cmake", "--build", "build", "-j2"]
```

Comment:
The `-j2` is supposed to be specific to the CI environment. I saw SciPy hardcoded some `-j2`s for parallel execution in some of its workflows on Linux and copied that. I considered using `$(nproc)` in CI. I'm not sure what the default should be. Locally I build with "-j24".